File: app/src/main/java/com/datarockets/mnchkn/activities/players/PlayersListView.java
package com.datarockets.mnchkn.activities.players;

import com.datarockets.mnchkn.models.Player;

import java.util.List;

public interface PlayersListView {
    void addPlayerToList(Player player);
    void deletePlayerFromList(int position);
    void setPlayersList(List<Player> players);
    void showAddNewPlayerDialog();
    void launchDashboard();
    void showStartContinueDialog();
    void showWarning();
}

File: app/src/main/kotlin/com/datarockets/mnchkn/activities/players/PlayersListView.kt
package com.datarockets.mnchkn.activities.players

import com.datarockets.mnchkn.models.Player

interface PlayersListView {
    fun addPlayerToList(player: Player)
    fun deletePlayerFromList(position: Int)
    fun setPlayersList(players: List<Player>)
    fun showAddNewPlayerDialog()
    fun launchDashboard()
    fun showStartContinueDialog()
    fun showWarning()
}